web/src/features/map-controls/MapControls.tsx

```diff
@@ -116,7 +116,7 @@ function DesktopMapControls() {
   const isConsumptionOnlyMode = useFeatureFlag('consumption-only');
 
   return (
-    <div className="pointer-events-none absolute right-3 top-2 z-20 hidden flex-col items-end md:flex">
+    <div className="pointer-events-none absolute right-3 top-2 z-20 mt-[env(safe-area-inset-top)] hidden flex-col items-end md:flex">
```

Comment:
nit: Should we consider adding a fallback value for all instances of `env(safe-area-inset-top)`?

Reply:
Not sure we need to, I only added it where we do a calc to be on the safe side.

But I can do it if we want to, previously we didn't have any though (when it was just for mobile).